nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/package.json

```diff
@@ -66,6 +66,7 @@
         "reset-css": "4.0.1",
         "roboto-fontface": "0.10.0",
         "rxjs": "6.5.2",
+        "sass": "^1.49.0",
```

Comment:
What is the reason for removing `node-sass` and adding `sass`? 

Also, the contributors in this project typically remove `^` or `~` from packages listed in `package.json` so that unintentional version bumps do not occur when running `npm install`.

Reply:
`node-sass` is deprecated. It's been replaced by `dart-sass` and the pure javascript implementation is contained in `sass`

I'll nix the `^`

Reply:
I've rebased and gotten it to complete the workflow. Thanks for your feedback